spec/features/workflow/publish_spec.rb

```diff
@@ -21,10 +53,12 @@ def when_i_visit_the_summary_page
   end
 
   def and_i_publish_the_edition
-    click_on "Publish"
-    choose I18n.t!("publish.confirmation.has_been_reviewed")
-    @content_request = stub_publishing_api_publish(@edition.content_id, update_type: nil, locale: @edition.locale)
-    click_on "Confirm publish"
+    travel_to(@publish_date = Time.current + 1.day) do
```

Comment:
What's the reason for needing to travel forward in time in order to publish?

Reply:
This is to ensure the date is distinct from other timestamps in the edition, but I've simplified it slightly by modifying `created_at` when the edition is created, so we're not adding a magic number when we freeze time.